### ISSUE TITLE
Surplus villagers

### ISSUE THREAD

**Issue by blablubbabcDEV** [Shopkeepers/Shopkeepers#21]
**Migrated from:** https://dev.bukkit.org/projects/shopkeepers/issues/11

**Originally posted by DemmyDemon (Aug 10, 2012):** 

What steps will reproduce the problem?
1. Place a shopkeeper inside what could be considered a large house.
2. Wait around for a long while.
3. The appropriate amount of population for that "large house" will spawn and walk around.What is the expected output? What do you see instead?
I expected Shopkeepers to block this behaviour. Alas, it did not.What version of the product are you using?
1.3Do you have an error log of what happened?
No log data available as far as I know.Please provide any additional information below.
Not sure if it's relevant, but I have only the ONE Shopkeeper, and it's a priest.

**Comment by blablubbabcDEV** [comment]
**Originally commented by DemmyDemon (Aug 11, 2012):** 

They WERE just appearing out of thin air, and in the same variation/robe as my Shopkeeper (Priest). 4 roaming "regular" testificates in priest robes.

I murdered them, and they haven't re-appeared yet.

I really have no idea how this happened in the first place, but Ive been running for about 4 hours with no sign of mysterious duplicates roaming around yet, so feel free to place under "Unable to reproduce" and close this.

It -might- have been caused by my foolishness with WorldEdit causing an out-of-memory crash shortly after placing the Shopkeeper?Â Â *shrug*

[update]

Going on 12 hours now, and no surplus villagersÂ Â O.o
            Edited Aug 11, 2012
